Source/UI/NewUI/BillingForm/View/BillingFormCellTextField.swift

```diff
@@ -9,13 +9,13 @@ protocol CellTextFieldDelegate: AnyObject {
     func textFieldShouldChangeCharactersIn(textField: UITextField, replacementString string: String)
 }
 
-final class CellTextField: UITableViewCell {
+final class BillingFormCellTextField: UITableViewCell {
```

Comment:
I would be quite keen for the `UITableViewCell` inheritance to be more obvious by keeping `cell` in the name.

Reply:
there's `Cell` in the `BillingFormCellTextField`